backend/homework/7/src/main/java/org/example/vehicleservice/services/VehicleService2.java

```diff
@@ -0,0 +1,54 @@
+package org.example.vehicleservice.services;
+
+import org.example.vehicleservice.constant.Constant;
+import org.example.vehicleservice.models.Speaker;
+import org.example.vehicleservice.models.Tyre;
+import org.example.vehicleservice.models.Vehicle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service("factory2")
```

Comment:
Use meaningful and descriptive names for your variables rather than using numerical suffixes.